src/app/components/core/scheduler/update-schedule/update-schedule.component.html

```diff
@@ -2,24 +2,33 @@
   <div class="modal-background"></div>
   <div class="modal-card">
     <header class="modal-card-head">
-      <p class="modal-card-title title-name">{{ rolesService.hasEditPermissions() ? 'Update Schedule' : 'View Schedule'
-        }}
+      <p class="modal-card-title title-name">
+        {{ rolesService.hasEditPermissions() ? 'Update Schedule' : 'View Schedule'}}
       </p>
-      <button class="delete" aria-label="close" (click)="toggleModal(false)"></button>
+      <button class="delete" aria-label="close" (click)="navToSchedulesPage()"></button>
     </header>
     <section class="modal-card-body">
-      <form [formGroup]="form" #f="ngForm" novalidate>
+      <form [formGroup]="form" novalidate>
         <ng-container *ngIf="rolesService.hasEditPermissions(); else staticForm">
           <div class="columns is-mobile">
             <div class="column">
               <div class="field">
                 <div class="form-group">
                   <label class="label is-small">Name</label>
-                  <p class="control is-fullwidth">
-                    <input type="text" class="input is-small" formControlName="name" placeholder="name" required
-                      trim="blur" title="No single quotes and double quotes!" [pattern]="QUOTATION_VALIDATION_PATTERN"/>
-                  </p>
-                  <small *ngIf="form.controls['name'].hasError('required') && form.controls['name'].touched" class="help is-danger level-left">Name is required</small>
+                  <ng-container *ngIf="schedule?.type?.name != 'STARTUP'; else staticName">
+                    <p class="control is-fullwidth">
+                      <input type="text" class="input is-small" formControlName="name" placeholder="name" required
+                        trim="blur" title="No single quotes and double quotes!"
+                        [pattern]="QUOTATION_VALIDATION_PATTERN" />
+                    </p>
+                    <small *ngIf="form?.controls['name']?.hasError('required') && form?.controls['name']?.touched"
+                      class="help is-danger level-left">Name is required</small>
```

Comment:
at other places, we use `*required`?

Reply:
Yes on many places like add filter, add task, create user, login etc.